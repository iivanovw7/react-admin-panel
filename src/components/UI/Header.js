import React, {Component} from 'react';


export const Header = (props) => {

  console.log(localStorage.getItem('name'));

  return (
    <div className={'header'}>
      <div>
        <i className="material-icons resizing">{props.icon}</i>
      </div>
      <div className={'title'}>
        <h3>{props.title}</h3>
      </div>
      <div style={{display: 'flex', alignItems: 'center'}}>
        <span style={{marginRight: '10px'}}>
          {localStorage.getItem('name')}
          </span>
        <button
          className={'header_button primary ripple'}
          onClick={() => {props.handleLogout()}}>Выйти</button>
      </div>
    </div>
  )

};