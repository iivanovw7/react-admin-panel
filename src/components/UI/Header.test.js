import React from 'react';
import { shallow } from 'enzyme';

import { Header } from './Header';

describe('Header', () => {

  it('should render correctly in "debug" mode', () => {

    const component = shallow(<Header debug />);

    expect(component).toMatchSnapshot();

  });

});