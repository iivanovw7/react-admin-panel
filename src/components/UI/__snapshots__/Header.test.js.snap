// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly in "debug" mode 1`] = `
<div
  className="header"
>
  <div>
    <i
      className="material-icons resizing"
    />
  </div>
  <div
    className="title"
  >
    <h3 />
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
      }
    }
  >
    <span
      style={
        Object {
          "marginRight": "10px",
        }
      }
    />
    <button
      className="header_button primary ripple"
      onClick={[Function]}
    >
      Выйти
    </button>
  </div>
</div>
`;
